core/app_main.c

```diff
@@ -21,6 +21,8 @@
 #include "esp/spi_regs.h"
 #include "esp/dport_regs.h"
 #include "esp/wdev_regs.h"
+#include "esp/wdt_regs.h"
+#include "esp/rtcmem_regs.h"
```

Comment:
Were these includes really needed? It was not giving any warnings last time I built? Might these be included by mistake, some other local changes?